tools/topology/topology2/cavs/CMakeLists.txt

```diff
@@ -28,6 +28,9 @@ PREPROCESS_PLUGINS=nhlt,NHLT_BIN=nhlt.bin"
 # CAVS SSP topology for TGL
 "cavs-nocodec\;cavs-tgl-nocodec\;NUM_DMICS=2,\
 PREPROCESS_PLUGINS=nhlt,NHLT_BIN=nhlt-cavs-tgl-nocodec.bin"
+# SSP topology for MTL
+"cavs-nocodec\;sof-mtl-nocodec\;PLATFORM=mtl,NUM_DMICS=2,\
+PREPROCESS_PLUGINS=nhlt,NHLT_BIN=nhlt-ace-mtl-nocodec.bin"
```

Comment:
what does NUM_DMICS mean? number of DMIC PCM capture or number of channels?

in the latter case, since it's a nocodec case why would we limit the number of channels?

Reply:
@plbossart it is channels. 4ch capture is currently broken on MTL. So I will change it when we have a fix.

Reply:
@juimonen do we have a PR pending to generate the NHLT data for this now ?

Reply:
@lgirdwood https://github.com/alsa-project/alsa-utils/pull/129handles the NHLT generation and it is already included in the CI docker image